Sage/carbon
# feat(classname prop): remove className from public props

The following properties have been removed from the listed components. Ensure any components
in the list below do not contain the associated properties.

BREAKING CHANGE:
- The `className` property has been removed from all components.
- Accordion: `scheme` and `buttonHeading`
- Box: `tabIndex`
- ButtonToggle: `grouped`
- Decimal: `onKeyPress`
- Icon: 'extra-small' variant
- Portrait: `gravatar`
- SimpleColorPicker: `isBlurBlocked`
- ListItem: `variant` (now inherits from parent List component)

### Current behaviour

Some of our components still support className as a prop, which would allow consumers to override the component's CSS styling, which is strongly discouraged for maintenance reasons.

### Checklist


#### QA


### Testing instructions

Ensure that all documentation is free of reference to `className`, ensure that passing custom class names to affected components does not change their styling.
